README.md

```diff
@@ -238,6 +238,7 @@ Deploy as Multitenant Application:
 - Run `cf deploy mta_archives/bookshop-mt_1.0.0.mtar`
 - Go to another subaccount in your global account, under subscriptions and subscribe to the application you deployed.
 - Run `cf map-route bookshop-mt-app <YOUR DOMAIN> --hostname <SUBSCRIBER TENANT>-<ORG>-<SPACE>-bookshop-mt-app` or create and bind the route manually.
+  - Example: `cf map-route bookshop cfapps.us10.hana.ondemand.com --hostname subscriber1-myOrg-mySpace-bookshop-mt-app`
```

Comment:
okay, maybe it would be better in that case to provide a description before the command about where to obtain the value for `<YOUR DOMAIN>`, e.g.:

```suggestion
  - `<YOUR DOMAIN>`: Find the app domain for your landscape by executing `cf domains`. It commonly starts with `cfapps.`
```

Reply:
Please have another look on the change. I have followed your suggestion and replaced the example with a description for the parameters that I struggled with during my setup.

Reply:
thanks 👍